src/snmpd/patch-5.7.3+dfsg/series

```diff
@@ -1,4 +1,5 @@
 0001-SNMP-Stop-spamming-logs-with-statfs-permission-denie.patch
 0002-at.c-properly-check-return-status-from-realloc.-Than.patch
-0003-CHANGES-BUG-2743-snmpd-crashes-when-receiving-a-GetN.patch
+#0003-CHANGES-BUG-2743-snmpd-crashes-when-receiving-a-GetN.patch
```

Comment:
Could you point me the reference that this patch is merged into latest version?

Reply:
Hi, just go to  line _409-411_ and _488-491_ in **table.c** file under `src/snmpd/net-snmp-5.7.3+dfsg/agent/helpers/` folder. The patch is merged there.